test/e2e/beta/metamask-beta-ui.spec.js

```diff
@@ -515,15 +515,15 @@ describe('MetaMask', function () {
       await driver.switchTo().window(dapp)
       await delay(regularDelayMs)
 
-      let contractStatus = await driver.findElement(By.css('#contractStatus'))
-      await driver.wait(until.elementTextMatches(contractStatus, /Deployed/))
+      let contractStatus = await driver.findElement(By.css('#contractStatus'), 10000)
```

Comment:
Can we use `await findElement(driver, By.css(...))` as that already has the longer timeout?

Reply:
Similarly for all instances below